packages/jaeger-ui/package.json

```diff
@@ -93,6 +93,7 @@
     "coverage": "npm run test-ci -- --coverage",
     "start:ga-debug": "REACT_APP_GA_DEBUG=1 REACT_APP_VSN_STATE=$(../../scripts/get-tracking-version.js) vite",
     "start": "vite",
+    "start:with-data": "REACT_APP_USE_SAMPLE_DATA=true vite",
```

Comment:
another option is to always enable this flag when running locally and display a dropdown in the graph with the name of test datasets (we could have more than one in the future). The graph wouldn't load the sample data by default, only if user selects it from the dropdown.